drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -4515,8 +4520,49 @@ void RasterizerSceneGLES3::render_scene(const Transform &p_cam_transform, const
 
 	//state.scene_shader.set_conditional( SceneShaderGLES3::USE_FOG,false);
 
-	if (use_mrt) {
+	// resolve buffers to textures
+	if (storage->frame.current_rt->buffers.active && use_mrt) {
```

Comment:
This blit should probably only take place when ``force_mrt`` is true. Otherwise these blits will take place every frame for everyone who uses mrts.